### ISSUE TITLE
Costumize shortcut of screenshots

### ISSUE THREAD

**Issue by OregSamSas** [MrAlex94/Waterfox#2956]
### **Is your feature request related to a problem? Please describe.**

The key shortcut of taking screenshot of a page used to have customizability under `about:addons > cog icon > Manage Extension Shortcuts`, as I remember, so I changed it to "Ctrl + Shift + Y" (conflicting with other hotkeys). But a few updates ago, it disappeared from plugins, and got migrated (or perhaps integrated), and so it disappeared from extension shortcuts, and the hotkey remained the default "Ctrl + Shift + S", and I feel I can't change it now. I even checked `about:config`, but I didn't find any appropriate option.

### **Describe the solution you'd like**

If someone suggested a method, with which I would be able to customize the shortcut again, I'd be happy, or if there isn't such an option, then please develop a new one for it.

### **Describe alternatives you've considered**

For me, it would be actually enough, at least to have an option to disable the keybind for taking screenshot, because I would still be able then to reach the function by pressing its button on the status bar.

### **Additional context**

_Nothing to show here._

**Comment by MrAlex94** [comment]
This might have been when Screenshots was a separate extension instead of being "built-in"? Had a quick look through the code and couldn't see anything that would allow you to change the shortcut, unfortunately. Sorry about that!

**Comment by OregSamSas** [comment]
> This might have been when Screenshots was a separate extension instead of being "built-in"

Yes, it appeared as a separate plugin under `about:addons > Plugins` that time.

> Had a quick look through the code and couldn't see anything that would allow you to change the shortcut, unfortunately. Sorry about that!

No problem, I'm used to experiencing the issue by now, so solving it isn't that urgent. 
Also, thank you for helping me so quickly, I was impressed for getting a response the very next day after I posted my issue.

**Comment by OregSamSas** [comment]
Good news: 
my issue with conflicting with other keys on a specific website has been resolved! I don't what had happened, but today I tried to use the shortcut again, and taking a screenshot didn't occur at all. Perhaps, the site made their shortcut to overwrite other matching shortcuts (I think it can be achieved via JavaScript, but I'm not sure).
So it isn't urgent for me anymore.
